torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -733,11 +731,14 @@ NvrtcFunction nvrtcCompile(
     // clamp down to register allocation granularity at warp level
     int effective_max_reg_per_warp = max_reg_per_warp /
         reg_allocation_granularity * reg_allocation_granularity;
-    max_register = static_cast<uint32_t>(
+    auto max_register = static_cast<uint32_t>(
         std::min(effective_max_reg_per_warp / warp_size, max_regs_per_thread));
+    // The maximum possible count allowed by ptxas is 255
```

Comment:
nitpick, we are not using `max_regs_per_thread` any more, we can wipe them out.

Reply:
Deleted